spec/factories/orchestration_template.rb

```diff
@@ -42,5 +42,6 @@
   factory :orchestration_template_vmware_cloud,
```

Comment:
@aiperon would you mind renaming this to `orchestration_template_vmware_cloud_with_content` to be consistent with other names?